app/models/concerns/searchable.rb

```diff
@@ -0,0 +1,67 @@
+module Searchable
+  extend ActiveSupport::Concern
```

Comment:
This concern is for "Search functions around Patients"?

Is something is both search-ish and, for example, notetaking-ish, which Concern does it belong in? Looks like Searchable?